app/views/home/index.html.erb

```diff
@@ -1,4 +1,34 @@
 <div class="container">
   <h1>Welcome to Imposter!</h1>
-  <h2>Coming Soon</h2>
-</div>
\ No newline at end of file
+  
+  <div class="rave-controls">
+    <table class="rave-table">
+      <tr>
+        <th>
+        
+        <div class="rave-status">
+          <% if @status == "on" %>
+            <p class="rave-on">ON</p>
```

Comment:
I agree as a rule you want to avoid overburdening your views with logic. But this is simple enough here that I'm ok with it as is.

The alternative is abstracting the logic into a model or perhaps in this case a service class. But that kind of aggressive refactoring can have its own problems as we've seen with Inside.